components/rhs_thread/rhs_thread.tsx

```diff
@@ -385,6 +386,8 @@ export default class RhsThread extends React.Component<Props, State> {
                     autoHide={true}
                     autoHideTimeout={500}
                     autoHideDuration={500}
+                    autoHeight={true}
+                    autoHeightMax={'calc(100vh - 190px - 56px - 62px)'}
```

Comment:
Could you put a comment somewhere here to explain where these numbers come from? I assume it's something like the channel header, comment box, and rhs header

Reply:
Indeed :D should have done that 👍